README.md

```diff
@@ -85,7 +97,7 @@ git clone https://github.com/fastplotlib/fastplotlib.git
 cd fastplotlib
 
 # install all extras in place
-pip install -e ".[notebook,docs,tests]"
+pip install -e ".[notebook,imgui]"
```

Comment:
I'm a dumbass I didn't notice this is actually in the "For Developers" section so yes you were right, put back `docs` and `tests`.

I need to go to bed, long day